demos_community.yaml

```
@@ -1,3 +1,17 @@
+- title: Hybrid Neural Network using Data-Reuploading technique
+  author: Nikolaos Schetakis (nikschet)
+  date: 11/04/2021
+  code: https://github.com/nsansen/Quantum-Machine-Learning/blob/main/Pennylane%20DEMO%20v4.ipynb
+
+  paper: https://arxiv.org/pdf/2111.03372.pdf
+  color: heavy-rain-gradient
+  description: |
+    By combining a standard Variational Classifier (VC) with a Data-Reuploading Classifier (DRC) 
+    we create a new Qnode which is then used as a quantum layer in a Hybrid Neural Network.
+    Note: this demo only works on Python versions 3.7-3.8.

```

```suggestion
    We combine a standard Variational Classifier with a Data-Reuploading Classifier,
    and integrate the resulting QNode as a quantum layer in a Hybrid Neural Network.
```

- I removed the acronyms, since they are never used 😆 
- I don't think the note needs to be included here, if needed it can be mentioned at the top of the notebook.